Added SEA so gun.user() would work

jmcudd opened the pull request:
I was getting this error when trying to create a new users with `this.$gun.user()`. Making this change seems to fix the issue. It looks `gun.user()` is dependent on SEA and it just wasn't being imported.
![screenshot_2018-07-14_23-34-12](https://user-images.githubusercontent.com/1785592/42730261-87482d50-87be-11e8-8a0d-3f592f2c7348.png)

Obviously the workaround for this is to just use your own instance of gun with the SEA import, but it wasn't obvious to me. I figured vue-gun's default configuration would just come with SEA, and the gunjs docs don't list any options that can be passed to Gun() to enabled SEA without an import of SEA.

If you prefer the workaround to importing SEA by default, then I suggest it be added to the README so users following the gunjs tutorials will  know that they have to import SEA in order to use `gun.user()`

```javascript
// Workaround
import Gun from 'gun';
import SEA from 'gun/sea';

var gun = Gun();
Vue.use(VueGun, {gun: gun});
```

sjones6 commented:
Thanks for digging through these issues, find this, and for the clear and helpful PR. This was built/released before SEA was around, hence why there's no mention of this, and I haven't come back to update the docs.

I think that your suggestion to add instructions is the best way to go here since the addition of 
`var SEA = require('gun/sea');` would add the SEA plugin to Gun in must bundlers even if that condition is not hit (i.e., they pass gun in). That way folks can still opt in to SEA if they want rather than enforcing it in vue-gun.

Do you mind updating your PR to remove that line and add your suggestions into the README?

jmcudd commented:
Yep. I added the instructions to the REAME.md, and removed the import from vue-gun.js. It looks like github is having some [issues](https://status.github.com/messages) though. The commit isn't showing up.

jmcudd commented:
Sorry for the delay, Github was throwing some errors when I pushed my changes to github this morning.

sjones6 commented:
No worries; I got stuck with Github being down too.

sjones6 commented:
Thanks for the PR! Much appreciated!
